src/bin/bat/clap_app.rs

```diff
@@ -133,47 +135,48 @@ pub fn build_app(interactive_output: bool) -> ClapApp<'static, 'static> {
     {
         app = app
                 .arg(
-                    Arg::with_name("diff")
+                    Arg::new("diff")
```

Comment:
CI is failing because this arg is conditionally present but is referenced by args that are unconditionally present